src/main/java/com/example/ajaxonboardingproject/dto/response/OrderResponseDto.kt

```diff
@@ -3,7 +3,7 @@ package com.example.ajaxonboardingproject.dto.response
 import java.time.LocalDateTime
 
 data class OrderResponseDto(
-        var id : Long,
-        var ticketIds : List<Long>,
-        var userId : Long,
+        var id : java.lang.Long,
```

Comment:
why you change all kotlin `Long` to java `Long`?

As we discussued earlier we wanted to try avoiding nullability for id field by using `lateinit`. But lateinit cannot be used with primitive types. 

Please all non-id fields back to kotlin `Long` or `Long?`